task_13/test/test.js

```diff
@@ -0,0 +1,45 @@
+const assert = require('chai').assert
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/jkalandarov/epam_stage_2/pullRequest?prid=8887248)
```suggestion
const assert = require("chai").assert
```